__tests__/createMap.test.dom.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @jest-environment jsdom
+ */
+import { Amplify } from "@aws-amplify/core"; 
+import { Geo } from "@aws-amplify/geo";
+import type { AmazonLocationServiceMapStyle } from "@aws-amplify/geo";
+ 
+import { createMap } from "../src/AmplifyMapLibreRequest";
+
+jest.mock("@aws-amplify/geo");
+
+describe('createMap', () => {
+  Amplify.Auth = {};
+  Amplify.Auth.currentCredentials = jest.fn().mockImplementation(() => {
+    return {
+      accessKeyId: "accessKeyId",
+      sessionToken: "sessionTokenId",
+      secretAccessKey: "secretAccessKey",
+      identityId: "identityId",
+      authenticated: true,
+      expiration: new Date(),
+    };
+  });
+
+  beforeEach(() => {
+    (Geo.getDefaultMap as jest.Mock).mockClear();
+  });
+
+  test("createMap returns a map object", async () => {
+    (Geo.getDefaultMap as jest.Mock).mockReturnValueOnce({
+      mapName: "",
+      region: "us-east-1",
+      style: "VectorEsriStreets"
```

Comment:
Using a `mapName` caused an error because maplibre instantiates a Worker whenever `mapName` is not an empty string. I can use a mock `mapName` by polyfilling `window.Worker` for the jsdom environment.